src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int sumSupply = 0;
+        int sumBuy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] lineSplit = line.split(",");
+                String operation = lineSplit[0];
+                int digit = Integer.parseInt(lineSplit[1]);
+                if (operation.equals("supply")) {
+                    sumSupply += digit;
+                } else if (operation.equals("buy")) {
+                    sumBuy += digit;
+                }
+            }
+        } catch (IOException exception) {
+            throw new RuntimeException("I can't perform the operation!", exception);
+        }
+        String report = createReport(sumSupply, sumBuy);
```

Comment:
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
```
`readFile` can return `String `or `List<String>` (your choice)

Reply:
i don`t quite understand what needs to be done